openshift-201/resource-mgmt.md

```diff
@@ -199,7 +204,7 @@ Set container cpu/memory request based on the above. The more accurately the req
 
 4. Set container cpu/memory limit.
 
-Setting a limit has the effect of immediately killing a container process or cpu throttling if the combined cpu/memory usage of all processes in the container exceeds the limit, and is therefore a mixed blessing. On the one hand, it may make unanticipated excess cpu/memory usage obvious early ("fail fast"); on the other hand it also terminates processes abruptly.
+Setting a limit has the effect of immediately killing a container process or cpu throttling if the combined cpu or memory usage of all processes in the container exceeds the limit, and is therefore a mixed blessing. On the one hand, it may make unanticipated excess cpu/memory usage obvious early ("fail fast"); on the other hand it also terminates processes abruptly.
```

Comment:
Hmmm. Ya I see your point. More I wanted to call out that the limit can also catch any bad code or mis-configuration that would cause memory or cpu to spike in the container so you catch it early and not like weeks down the road do you find a memory leak bug as the limit was too high. Also calling out that setting a limit will start killing processes or throttling things if set too low.

Happy for a suggestion here though!

Reply:
Wondering if there is a way of describing this with a 'goldilocks' example, might help to emphasize a happy middle ground?:
- If the limit is set too low =... 
- If the limit is set too high/not set =... 
- If the limit is set _just right_ =...

Reply:
Great idea! I added a summary to this section. Let me know if that works.

Reply:
Awesome, thanks Phil. Looks good to me!